### DISCUSSION TITLE
Can't convert properly graph tool graph properties (graph label)

### DISCUSSION THREAD

**Discussion by turexx** [pyg-team/pytorch_geometric#9226] [Q&A]
I'm trying to convert my graphs data from graph tool to pyg, and actually it works. But when I map the graphs' label in pyg, the structure of Data object is not correct. After the label map I obtain something like that:
`Data(x=[50000], edge_index=[2, 199992], y=1)`
As you can see, y is not a tensor, even if I convert it to a tensor, it is still y=1 without square brackets.

**Comment by rusty1s** [comment]
This is what I get (which is expected behavior):
```python
import torch

from torch_geometric.data import Data

data = Data(y=2.0)
print(data)
data = Data(y=torch.tensor(2.0))
print(data)
data = Data(y=torch.tensor([2.0]))
print(data)
```
```
Data(y=2.0)
Data(y=2.0)
Data(y=[1])
```
Note that if we display, we unwrap the element for zero-element tensors.
